apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js

```diff
@@ -17,7 +17,7 @@ async function sendBillingReceiptNotificationsWeekends () {
 }
 
 /** PODS work according to UTC, so time in cron-tasks should also be according to UTC too. */
-const sendBillingReceiptNotificationsWorkDaysTask = createCronTask('sendBillingReceiptNotificationsWorkDaysTask', '17 7-12 * * 1-5', sendBillingReceiptNotificationsWorkDays)
```

Comment:
Do not forget to clear crontask within bull. As I remember, crontasks disappeared in code will not removed automatically.